testsuite/MDAnalysisTests/guesser/test_base.py

```diff
@@ -33,76 +33,126 @@
 from MDAnalysis import _TOPOLOGY_ATTRS, _GUESSERS
 
 
-class TestBaseGuesser():
+class TestBaseGuesser:
 
     def test_get_guesser(self):
         class TestGuesser1(GuesserBase):
-            context = 'test1'
+            context = "test1"
 
         class TestGuesser2(GuesserBase):
-            context = 'test2'
+            context = "test2"
 
-        assert get_guesser(TestGuesser1).context == 'test1'
-        assert get_guesser('test1').context == 'test1'
-        assert get_guesser(TestGuesser2()).context == 'test2'
+        assert get_guesser(TestGuesser1).context == "test1"
+        assert get_guesser("test1").context == "test1"
+        assert get_guesser(TestGuesser2()).context == "test2"
 
     def test_get_guesser_with_universe(self):
         class TestGuesser1(GuesserBase):
-            context = 'test1'
+            context = "test1"
 
         u = mda.Universe.empty(n_atoms=5)
         guesser = get_guesser(TestGuesser1(), u, foo=1)
 
         assert len(guesser._universe.atoms) == 5
-        assert 'foo' in guesser._kwargs
+        assert "foo" in guesser._kwargs
 
     def test_guess_invalid_attribute(self):
-        with pytest.raises(ValueError,
-                           match='default guesser can not guess '
-                                 'the following attribute: foo'):
-            mda.Universe(datafiles.PDB, to_guess=['foo'])
+        with pytest.raises(
+            ValueError,
+            match="default guesser can not guess "
+            "the following attribute: foo",
+        ):
+            mda.Universe(datafiles.PDB, to_guess=["foo"])
 
     def test_guess_attribute_with_missing_parent_attr(self):
-        names = Atomnames(np.array(['C', 'HB', 'HA', 'O'], dtype=object))
+        names = Atomnames(np.array(["C", "HB", "HA", "O"], dtype=object))
         masses = Masses(
-            np.array([np.nan, np.nan, np.nan, np.nan], dtype=np.float64))
-        top = Topology(4, 1, 1, attrs=[names, masses, ])
-        u = mda.Universe(top, to_guess=['masses'])
-        assert_allclose(u.atoms.masses, np.array(
-            [12.01100, 1.00800, 1.00800, 15.99900]), atol=0)
+            np.array([np.nan, np.nan, np.nan, np.nan], dtype=np.float64)
+        )
+        top = Topology(
+            4,
+            1,
+            1,
+            attrs=[
+                names,
+                masses,
+            ],
```

Comment:
```suggestion
            ]
```
Trailing comma messing things up.